tests/upgrades/test_bookmarks.py

```diff
@@ -0,0 +1,96 @@
+"""Test for bookmark related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+from upgrade_tests import post_upgrade
+from upgrade_tests import pre_upgrade
+
+from robottelo.constants import BOOKMARK_ENTITIES
+from robottelo.test import APITestCase
+
+
+BOOKMARK_ENTITIES = list(BOOKMARK_ENTITIES)
```

Comment:
```
In [3]: from robottelo.constants import BOOKMARK_ENTITIES                                                                                                                                     

In [4]: type(BOOKMARK_ENTITIES)                                                                                                                                                               
Out[4]: list

```
I think, no need to cast again. Not sure what was the intention

Reply:
yes it not required.